web/portal/src/main/webapp/WEB-INF/conf/organization/idm-externalstore-configuration.xml

```diff
@@ -38,6 +38,11 @@
         <description>Cron expression used to schedule the job that will import periodically data from external store</description>
         <value>${exo.idm.externalStore.import.cronExpression:0 59 23 ? * *}</value>
       </value-param>
+			<value-param>
```

Comment:
As I understand, when enabling MFA, we need to set manually this property too ? couldn't it be activated automatically ?